bpfman/src/multiprog/xdp.rs

```diff
@@ -49,6 +51,62 @@ pub struct XdpDispatcher {
 }
 
 impl XdpDispatcher {
+    pub(crate) fn get_test(
```

Comment:
Should we name the test dispatchers something other than "xdp_dispatcher" and "tc_dispatcher"?  E.g., "bpfman_xdp_verify" and "bpfman_tc_verify"?  I loaded and unloaded some tc and xdp programs and was surprised to see dispatchers remaining.  Also, when the user runs `sudo bpftool prog list` it makes it easier to differentiate between real dispatchers and the test ones.

Reply:
#1427